inventory/host_vars/www.openfoodnetwork.org.uk/config.yml

```diff
@@ -7,8 +7,6 @@ rails_env: production
 unicorn_workers: 4
 unicorn_timeout: 720
 
-swapfile_size: 1G
-
```

Comment:
but will that run `swapoff` and remove all the stuff?

Reply:
You already removed the swap, right?

This Ansible role is not capable of removing or decreasing swap... :disappointed:

Reply:
Yes, I manually did a swapoff. Could you please add an issue for it?